bftclient/include/bftclient/bft_client.h

```diff
@@ -44,7 +44,7 @@ class Client {
   // Throws a BftClientException on error.
   Reply send(const WriteConfig& config, Msg&& request);
   Reply send(const ReadConfig& config, Msg&& request);
-  std::deque<Reply> sendBatch(std::deque<WriteRequest>& write_requests, const std::string& cid);
+  std::unordered_map<uint64_t, Reply> sendBatch(std::deque<WriteRequest>& write_requests, const std::string& cid);
```

Comment:
std::unordered_map<uint64_t, Reply> is too long, create define for it.

Reply:
Added typedef called ReplySeqNumMap